web/server/vue-cli/src/views/NewFeatures.vue

```diff
@@ -2,6 +2,28 @@
   <!-- eslint-disable max-len -->
   <v-container fluid>
     <v-timeline align-top>
+      <v-timeline-item fill-dot icon="mdi-star" color="green lighten-1">
+        <new-release-item color="green lighten-1">
+          <template v-slot:title>
+            <a
+              href="https://github.com/Ericsson/codechecker/releases/tag/v6.22.1"
+              target="_blank"
+              class="white--text"
+            >
+              Highlights of CodeChecker 6.22.1 release
+            </a>
+          </template>
+
+          <new-feature-item>
+            <template v-slot:title>
+              Bugfix, related to CodeChecker server
```

Comment:
I don't know, I'm not too excited about this. Maybe there isn't anything to highlight this release -- fixing a bug is important, but maybe this is not the page for its celebration.

Reply:
We already released this highlights page as is in v6.22.1. In a subsequent pull request, we may change it to better represent the patch release.